modules/custom/d_content/config/update/d_content.8118.yml

```diff
@@ -0,0 +1,12 @@
+core.entity_form_display.node.content_page.default:
+  add:
+    content:
+      field_header_layout:
+        region: content
+        type: options_select
```

Comment:
too, jeżeli musi być zmienione, to powinno być uwzględnione w  `d_content.8116.yml`, ale widzę, że już jest:
![image](https://user-images.githubusercontent.com/37533838/88646118-1fde0480-d0c5-11ea-9830-1d6655bd18dc.png)